### PR TITLE
Has values class

### PR DESCRIPTION
**Pull request by thijsvdanker**
**tldr;** This PR adds a `.multiselect--has-values` class to the main element when there is one or more items selected.

## Reason
My placeholder is pretty long, so I need to set a width for the input.
The width for the input is currently controlled in the `inputStyle()` method here https://github.com/shentao/vue-multiselect/blob/master/src/Multiselect.vue#L256.

By adding the `.multiselect-has-values` class, I am able to mimic the default behaviour (showing the placeholder when there are no values, or the select is active; hiding it in other cases.

I'm using this snippet of sass
```scss
.multiselect,
.multiselect.multiselect--active.multiselect--has-values {
  .multiselect__input {
    width: 300px !important;
  }
}
.multiselect.multiselect--has-values {
  .multiselect__input {
    width: 0 !important;
  }
}
```

Curious to hear if there's a better way to do this.

### CODE REVIEW

**Comment by thijsvdanker** [comment]
I think I have an easier solution in https://github.com/shentao/vue-multiselect/pull/606 where you can just set the `placeholderWidth' property.
